crates/turborepo-lib/src/opts.rs

```diff
@@ -43,25 +43,36 @@ pub enum Error {
     Config(#[from] crate::config::Error),
 }
 
-/// The fully resolved options for Turborepo. This is the combination of config,
-/// including all the layers (env, args, etc.), and the command line arguments.
 #[derive(Debug, Clone)]
-pub struct Opts {
-    pub root_turbo_json_path: AbsoluteSystemPathBuf,
+pub struct APIClientOpts {
     pub api_url: String,
     pub timeout: u64,
     pub upload_timeout: u64,
     pub token: Option<String>,
     pub team_id: Option<String>,
     pub team_slug: Option<String>,
+    pub login_url: String,
+    pub preflight: bool,
+}
+
+#[derive(Debug, Clone)]
+pub struct ConfigOpts {
```

Comment:
Names are hard, but I don't think we should name this `ConfigOpts`: `ConfigurationOptions` populate an `Opts` struct which contains `ConfigOpts` is confuses me. Maybe `RepoOpts` since package manager/`turbo.json` are related to the whole repo?